src/asm/parser.y

```diff
@@ -1463,8 +1463,11 @@ relocexpr_no_str : scoped_anon_id { rpn_Symbol(&$$, $1); }
 			lexer_ToggleStringExpansion(false);
 		} T_LPAREN scoped_anon_id T_RPAREN {
 			struct Symbol const *sym = sym_FindScopedSymbol($4);
+			uint32_t value = sym_IsPC(sym) ? !!sect_GetSymbolSection()
+					: sym_Is_NARG(sym) ? !!macro_GetCurrentArgs()
+					: !!sym;
 
-			rpn_Number(&$$, !!sym);
+			rpn_Number(&$$, value);
```

Comment:
That's fine and dandy, but what about `ISCONST` and other builtin functions, for example? This seems iffy to me.

Reply:
Actually you're right: not rgbasm functions, but other parts of the code, should consider these symbols undefined. For example, `"{_NARG}"` should be able to report "Interpolated symbol "_NARG" does not exist".

I think it would work to edit `sym_FindExactSymbol` to potentially return `NULL` for those two symbols.